### PR TITLE
fix RPM package repository url

### PR DESCRIPTION
**Pull request by soubinan**
Add a comment for RHEL installation

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
@blueorangutan docbuild

**Comment by rohityadavcloud** [comment]
@blueorangutan docbuild

**Comment by rohityadavcloud** [comment]
@blueorangutan docbuild

**Comment by rohityadavcloud** [comment]
@blueorangutan docbuild

File: source/installguide/management-server/_pkg_repo.rst [hunk 1]
```diff
@@ -51,11 +51,13 @@ To add the CloudStack repository, create
 ``/etc/yum.repos.d/cloudstack.repo`` and insert the following
 information.
 
+In the case of RHEL you could replace centos by rhel in baseurl
```
Comment (DaanHoogland) [inline source/installguide/management-server/_pkg_repo.rst:54]:
```suggestion
In the case of RHEL being used, you can replace 'centos' by 'rhel' in the value of baseurl
```

**Review by DaanHoogland — APPROVED** [review]
thanks @soubinan , looks good, just an optional sugestion for improvement.

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Pull request merged**